glsl/src/transpiler/spirv.rs

```diff
@@ -41,7 +72,7 @@ pub fn transpile_translation_unit<F>(
   f: &mut F,
   tu: &syntax::TranslationUnit,
   kind: ShaderKind
-) -> Result<(), String> where F: Write {
+) -> Result<(), String> where F: std::fmt::Write {
```

Comment:
Why not directly keep the `use` and use `Write`? here?

Reply:
The binary version of the function uses `std::io::Write` instead of `std::fmt::Write`, so I didn't want the imports to be confusing